src/main/webapp/app/exam/participate/exam-cover/exam-participation-cover.component.html

```diff
@@ -15,86 +15,86 @@ <h2 class="col-md-6" style="text-align: center; font-weight: normal" id="exam-fi
             </span>
             <br />
         </h2>
-    }
-    @if (startView) {
-        <h2 class="col-md-6" style="text-align: center; font-weight: normal">
-            <span>
-                {{ 'artemisApp.examParticipation.greeting' | artemisTranslate: { title: exam.title } }}
-            </span>
-            <br />
-        </h2>
-    }
-    @if (!startView && !studentFailedToSubmit) {
         <jhi-exam-timer class="col-md-3 justify-content-end" [isEndView]="true" [endDate]="graceEndDate" [criticalTime]="criticalTime" />
-    }
-    @if (startView) {
-        <div class="col-md-3"></div>
-    }
-</div>
-@if (!startView && !studentFailedToSubmit) {
+    </div>
     <div class="w-100 d-flex">
         <div class="col-md-3"></div>
         <p class="col-md-6 font-weight-bold" style="text-align: center">
             <span jhiTranslate="artemisApp.examParticipation.submitFinalExam"></span>
         </p>
     </div>
 }
-<div class="container d-flex flex-column align-items-center justify-content-center">
-    @if (startView) {
-        <jhi-exam-general-information [exam]="exam" [studentExam]="studentExam" [displayOnExamCover]="true" />
-    }
+@if (!startView) {
     <div><span style="display: table; margin-left: auto; margin-right: auto" [innerHTML]="formattedGeneralInformation"></span></div>
-    @if (studentFailedToSubmit) {
-        <div class="mb-2 font-weight-bold text-danger">
-            {{ 'artemisApp.studentExam.submissionNotInTime' | artemisTranslate }}
-        </div>
-    }
-    @if (!studentFailedToSubmit) {
-        <div class="form-check mt-1 ps-0">
-            <input
-                [(ngModel)]="confirmed"
-                type="checkbox"
-                id="confirmBox"
-                (click)="updateConfirmation()"
-                class="form-check-input"
-                [class.ms-0]="!(startView ? this.exam.confirmationStartText : this.exam.confirmationEndText)"
-                [required]="inserted"
-                [disabled]="startView ? waitingForExamStart : false"
-            />
-            <label for="confirmBox" id="formatted-confirmation-text" class="form-check-label" [innerHTML]="formattedConfirmationText"></label>
+}
+@if (studentFailedToSubmit) {
+    <div class="col-md-3">
+        <jhi-exam-live-events-button />
+    </div>
+    <div class="d-flex justify-content-between">
+        <div class="col-12 text-center my-auto">
+            <h3>{{ exam.title }}</h3>
+            <div class="mb-2 font-weight-bold text-danger">
+                {{ 'artemisApp.studentExam.submissionNotInTime' | artemisTranslate }}
```

Comment:
@pzdr7 Suggested to use a span with jhiTranslate. In order to make it consistent, could we use it here as well?